### DISCUSSION TITLE
is there any way of using a civitai style/model on fooocus?

### DISCUSSION THREAD

**Discussion by dpv20** [lllyasviel/Fooocus#1736] [Q&A]
thats it, have anyone implement it? im just starting with IA pictures and i want to use this as im a huge fan of one piece, but i dont have any idea where do i start to manage to implement it

https://civitai.com/models/4219/wano-style-lora

**Comment by eddyizm** [comment]
yeah, you just have to download it and put it in the folder. 
Although, someone may be able to correct me, the one piece one that you linked may not work because from the readme

> Fooocus support SDXL models on Civitai

and the one you linked is an SD.

**Reply by eddyizm** [reply]
quick follow up, it seems like that SDXL is only for the base model, you may be able to use the lora and refiner as SD 1.5
![image](https://github.com/lllyasviel/Fooocus/assets/12236892/27911a49-f263-4568-ba28-352522da1b40)

**Reply by JT-Gresham** [reply]
Um..the OP linked to a LoRA...not a model...or a refiner...it's a LoRA so I think they just need to download it, toss it into the lora folder, activate it in the settings...set the weight...and use the keywords in the prompt to trigger it (hopefully).

Someone might be able to correct me as well and please do if I got it wrong.

**Reply by eddyizm** [reply]
@JT-Gresham I think you are right and I may have been too eager to give an answer. Thanks for clarifying!

**Comment by dpv20** [comment]
yeah man i realice yesterday, i need to put it in the lora folders

i havent test it, but i supuouse i need to download the models like [Midnight Mixer Melt] that they say in the description

**Comment by dpv20** [comment]
it says that i should use a .ckpt, anyone know where do i put those?
is this
https://huggingface.co/hakurei/waifu-diffusion-v1-4/blob/main/vae/kl-f8-anime2.ckpt

**Reply by JT-Gresham** [reply]
Hmm..well, that's a VAE and I think Fooocus has it's own custom VAE.  There's a vae folder in the models folder, but I've not ever used a model that needed one.  The other's I've used had them baked into the base model, but if it's possible to use with Fooocus, it'd be a safe bet to drop it in the vae folder, I'd think.  No idea if it'll just load for you when you load the base model though...it might.  Try it and let me know what you find out.

Re-Edit: I just looked at it a bit closer, you might want to have a look in the "Discussion" section...I think I saw a .safetensors version mention when I scanned it.  I didn't see it listed as SDXL anywhere...if it's not, the model won't load in Fooocus as a base model.

**Comment by dpv20** [comment]
the mayority are not working, or its a diference when its says is a CHECKPOINT MERGE
for example

Midnight Mixer Melt.safetensors (https://civitai.com/models/49976/drbob2142-midnight-mixer-melt)
photon_v1.safetensors (https://civitai.com/models/84728?modelVersionId=90072)
realcartoon3d_v13.safetensors (https://civitai.com/models/94809?modelVersionId=276583)

this did work and is the same supousely
photopediaXL_45.safetensors (https://civitai.com/models/189109?modelVersionId=259323)


i got 
NotImplementedError: You have selected base model other than SDXL. This is not supported yet.

fooocus only suuports XL models?

**Reply by docppp** [reply]
as a base model, yes.

**Reply by dpv20** [reply]
so i can only use XL models? do you have any suggestion to what i could download?

**Reply by eddyizm** [reply]
When you are there you can filter by sdxl. There's still plenty.

**Reply by JT-Gresham** [reply]
Fooocus is really pretty straight forward with most of what you need to know.

1. Base models need to be SDXL, like it says...so make sure to get SDXL by using the filter on civitai like eddy said.  I wouldn't even touch a model that isn't in .safetensors format. LoRAs are also in .safetensors format so don't get them mixed up.
2. Refiners can be SD1.5 or SDXL...either works fine as refiners but read up on the base model. Many state outright that you don't need a refiner and will usually give you good info for setup to get the best performance. NOTE: There's a checkbox at the bottom of the Advanced tab for "Developer Debug Mode" that will open up a good number of options...but use at your own risk.
3. It seems to me most LoRAs work...I haven't messed with any that are marked otherwise (like Lycoris) Someone else might be able to provide insight to those.  The LoRAs will often have keywords listed and you need to use them in your prompt to get the LoRA to trigger. Setting the weights is also important depending on how the LoRA was compiled...that info is also often noted in the LoRA's info page.

That's basically it to get running with good results.  The rest (and probably the most important) is creating great prompts..both positive and negative.  Styles are also very useful so don't overlook them.  I've even created a whole slew of custom styles, but that's a little more involved and I mainly use them for convenience.  Easy to understand just by looking at the style's .json file in a text editor.  HAPPY FOOOCUSING!
